packages/salesforcedx-utils-vscode/test/unit/i18n/localizationTest.ts

```diff
@@ -6,14 +6,9 @@
  */
 
 import { expect } from 'chai';
-import {
-  BASE_FILE_EXTENSION,
-  BASE_FILE_NAME,
-  Config,
-  DEFAULT_LOCALE,
-  Localization,
-  Message
-} from '../../../src/i18n/localization';
+import { Config, Localization, Message } from '../../../src/i18n';
```

Comment:
Updated the test to remove the dynamic `requires` since that breaks webpack's dependency navigation.